.github/hilbert-integration-check-failure-issue.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Hilbert integration test failed
+assignees: aradnaevinf
```

Comment:
```suggestion
assignees: aradnaevinf, singular-value, SalahedeenIssa
```

(Since @SalahedeenIssa and I are doing more Hilbert stuff)

Reply:
Done.